packages/gsa/PKGBUILD

```
@@ -2,41 +2,36 @@
 # See COPYING for license details.
 
 pkgname=gsa
-pkgver=22.4
-pkgrel=3
-pkgdesc='Greenbone Security Assistant.'
-arch=('x86_64' 'aarch64')
-url='https://github.com/greenbone/gsa'
+pkgver=22.4.0
+pkgrel=2
```

Else it will be considered as older?

```suggestion
pkgrel=4
```